ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2017.2.0f2
+m_EditorVersion: 2017.2.0f1
```

Comment:
![image](https://user-images.githubusercontent.com/13334553/31206885-80fed638-a92f-11e7-9739-8a4f6f086ead.png)

Reply:
Easier just to revert all changes